### PR TITLE
Update FAQ (#505)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #505 to release-1.0

---

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/faq.md [hunk 1]
```diff
@@ -149,3 +149,43 @@ DM 在最后 `rename ghost_table to origin table` 的步骤会把内存的 DDL 
 4. 通过 `start-task` 启动迁移任务。
 
 5. 通过 `query-status` 观察迁移任务状态，当 `syncerBinlog` 超过 `checkpoint-T` 与 `checkpoint-S` 中的较大值后（在本例中，为 `(mysql-bin.000100, 1234)`），即可还原 `safe-mode` 为原始值并重启迁移任务。
+<<<<<<< HEAD
```
Comment (yikeke) [inline zh/faq.md:152]: Please help resolve the conflicts. @lance6716

File: zh/faq.md [hunk 2]
```diff
@@ -149,3 +149,40 @@ DM 在最后 `rename ghost_table to origin table` 的步骤会把内存的 DDL 
...
+
+## 2.0 集群运行 1.0 已有数据迁移任务时报错 `Error 1054: Unknown column 'binlog_gtid' in 'field list'`
+
+在 DM 2.0 中，为 checkpoint 等元信息表引入了更多的字段。如果在 2.0 中，通过 `start-task` 直接使用 1.0 集群的任务配置文件从增量复制阶段继续运行，则会出现 `Error 1054: Unknown column 'binlog_gtid' in 'field list'` 错误。
+
+对于此错误，可使用以下任一方式进行处理：
+
+- [使用 TiUP 将 DM 1.0 集群导入到全新的 2.0 集群](maintain-dm-using-tiup.md#导入-dm-ansible-部署的-dm-10-集群并升级)。
+- [手动将 DM 1.0 的数据迁移任务导入到 2.0 集群](manually-upgrade-dm-1.0-to-2.0.md)。
+
```
Comment (yikeke) [inline zh/faq.md:170]:
```suggestion
```
这些行在 v1.0 里本来就不存在，应该不用加？ @lance6716

File: zh/faq.md [hunk 3]
```diff
@@ -149,3 +149,40 @@ DM 在最后 `rename ghost_table to origin table` 的步骤会把内存的 DDL 
...
+## DM 分库分表同步中没有明显报错，但是下游数据丢失
+
+需要检查配置项 `block-allow-list` 和 `table-route`：
+
+- `block-allow-list` 填写的是上游数据库表，可以在 `do-tables` 前通过加 “~” 来进行正则匹配。
+- `table-route` 不支持正则，采用的是通配符模式，所以 `table_parttern_[0-63]` 只会匹配 table_parttern_0 到 table_pattern_6 这 7 张表。
+
+## DM 上游无写入，replicate lag 监控无数据
+
+在 DM v1.0 中，需要开启 `enable-heartbeat` 才会产生该监控数据。v2.0 中，尚未启用该功能，replicate lag 监控无数据是预期行为。
```
Comment (yikeke) [inline zh/faq.md:188]:
```suggestion
在 DM v1.0 中，需要开启 `enable-heartbeat` 才会产生该监控数据。
```

File: zh/faq.md [hunk 4]
```diff
@@ -149,3 +149,40 @@ DM 在最后 `rename ghost_table to origin table` 的步骤会把内存的 DDL 
...
+## 2.0 集群运行 1.0 已有数据迁移任务时报错 `Error 1054: Unknown column 'binlog_gtid' in 'field list'`
+
+在 DM 2.0 中，为 checkpoint 等元信息表引入了更多的字段。如果在 2.0 中，通过 `start-task` 直接使用 1.0 集群的任务配置文件从增量复制阶段继续运行，则会出现 `Error 1054: Unknown column 'binlog_gtid' in 'field list'` 错误。
+
+对于此错误，可使用以下任一方式进行处理：
+
+- [使用 TiUP 将 DM 1.0 集群导入到全新的 2.0 集群](maintain-dm-using-tiup.md#导入-dm-ansible-部署的-dm-10-集群并升级)。
+- [手动将 DM 1.0 的数据迁移任务导入到 2.0 集群](manually-upgrade-dm-1.0-to-2.0.md)。
+
+## TiUP 无法部署 DM 的某个版本（如 v2.0.0-hotfix）
```
Comment (yikeke) [inline zh/faq.md:171]:
```suggestion
## TiUP 无法部署 DM 的某个版本
```
这里能举个 v1.0 版本的例子吗

**Comment by lance6716** [comment]
@yikeke 感觉这个不用cherry-pick到1.0了，这个PR主要是2.0的问题，而且新用户应该也都是2.0开始用了

**Comment by yikeke** [comment]
> @yikeke 感觉这个不用cherry-pick到1.0了，这个PR主要是2.0的问题，而且新用户应该也都是2.0开始用了

可以的，FYI @csuzhangxc

**Pull request closed without merging**